4.4/4.4.6/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.6
+Создать объект со следующей логикой:
+Появляется сообщение: "Введите любой текст (он будет накапливаться)"
+Потом появляется конферм: "Накопленный текст: "[... ... ...]". Хотите еще добавить?"
+Когда пользователь нажмет ОК - начинается всё сначала, появляется первое сообщение.
+Если нажимает ОТМЕНА - всё заканчивается и выводится просто алерт: "Накопленный текст: "[... ... ...]""
+Каждый новый добавленнный текст отделяется от предыдущего с помощью пробела.
+<script>
+	const userTextObj = {
+		text: '',
+		isContinue: true,
```

Comment:
let's discuss this once again ) we can simplify this logic, we can create it more correct